AirCasting/Utils/FileLineReader/DefaultFileLineReader.swift

```diff
@@ -22,4 +22,23 @@ class DefaultFileLineReader: FileLineReader {
         fclose(file)
         progress(.endOfFile)
     }
+    
+    func readLastNonEmptyLine(of fileURL: URL) throws -> String? {
+//        assert(!Thread.isMainThread)
```

Comment:
no commented out code plz 🧹

Reply:
Do you have an idea how we can handle this so that it doesn't crash when running tests?